wxz-validator.php

```diff
@@ -59,5 +69,24 @@ function raise_warning( $code, $message ) {
 				continue;
 			}
 		}
+
+		if ( 'users' === $type ) {
+			$result = $validator->validate( Helper::toJSON( $item ), 'https://wordpress.org/schema/user.json' );
```

Comment:
We can just use the file content, no?
```suggestion
			$result = $validator->validate( $file['content'], 'https://wordpress.org/schema/user.json' );
```

Reply:
Nope, per [the docs](https://opis.io/json-schema/2.x/php-validator.html#data-types), the data can't be a JSON-encoded string.

Reply:
Ah, right. But just using `$item` should work as per
```
// decoding a JSON string
$data = json_decode('{"name": "opis"}');
```

```suggestion
			$result = $validator->validate( $item, 'https://wordpress.org/schema/user.json' );
```